### ISSUE TITLE
Add occupation details

### ISSUE THREAD

**Issue by clemens-msupply** [openmsupply/open-msupply#1426]
Currently we only have a single occupation field in the our generic data model.

We like to record a occupation group/status and a more detailed occupation entry though, e.g.:

- Formally employed
- Software developer

**Comment by richardmoizeau** [comment]
Yes, idea is to keep the DDL with standard values (eg. `Formally Employed`) and to add a **second** free text field to enter the patient's occupation (eg. `Police Officer`).
